test/test_security_service.py

```diff
@@ -97,7 +97,7 @@ def test_vm_firewall_properties(self):
 
     @helpers.skipIfNoService(['security.vm_firewalls'])
     def test_crud_vm_firewall_rules(self):
-        name = 'cb-crudfw_rules-{0}'.format(helpers.get_uuid())
+        name = 'cb-crudfw-rules-{0}'.format(helpers.get_uuid())
```

Comment:
Just a note: May need a corresponding change in validation regex here: https://github.com/gvlproject/cloudbridge/blob/eb20e3d88f2893a6e281ceead7ab8fd3527be6af/cloudbridge/cloud/base/resources.py#L69

Or more appropriately, the GCE resources that do not support dashes can selectively override the CB_NAME_PATTERN property to implement the appropriate regex?

Reply:
Bai is going to make GCE resources override the pattern in his next PR.

Reply:
Sounds good.